prowler/lib/outputs/ocsf/ocsf.py

```diff
@@ -0,0 +1,198 @@
+import os
+import sys
+from copy import deepcopy
+from typing import List
+
+from py_ocsf_models.events.base_event import SeverityID, StatusID
+from py_ocsf_models.events.findings.detection_finding import DetectionFinding
+from py_ocsf_models.events.findings.detection_finding import (
+    TypeID as DetectionFindingTypeID,
+)
+from py_ocsf_models.events.findings.finding import ActivityID, FindingInformation
+from py_ocsf_models.objects.account import Account, TypeID
+from py_ocsf_models.objects.cloud import Cloud
+from py_ocsf_models.objects.container import Container
+from py_ocsf_models.objects.group import Group
+from py_ocsf_models.objects.metadata import Metadata
+from py_ocsf_models.objects.organization import Organization
+from py_ocsf_models.objects.product import Product
+from py_ocsf_models.objects.remediation import Remediation
+from py_ocsf_models.objects.resource_details import ResourceDetails
+
+from prowler.lib.logger import logger
+from prowler.lib.outputs.finding import Finding
+from prowler.lib.outputs.output import Output
+
+
+class OCSF(Output):
+    """OCSF class that transforms the findings into the OCSF format.
+
+    Args:
+        Output (ABC): Abstract class that defines the methods that the child classes should implement.
+    """
+
+    def transform(self, findings: List[Finding]) -> None:
+        """Transforms the findings into the OCSF format.
+
+        Args:
+            findings (List[Finding]): a list of Finding objects
+        """
+        try:
+            for finding_output in findings:
+                finding = deepcopy(finding_output)
```

Comment:
I think you don't need this copy as far as you are not modifying the `finding` object.

Reply:
True, done